tests/components/plugwise/test_number.py

```diff
@@ -27,7 +23,7 @@ async def test_anna_max_boiler_temp_change(
 ) -> None:
     """Test changing of number entities."""
     await hass.services.async_call(
-        NUMBER_DOMAIN,
+        Platform.NUMBER,
```

Comment:
I think it's been discussed before with @MartinHjelmare 
This is wrong => services use domains, not platforms

Reply:
Ok, I was not aware of this. I thought there were the same.
Would be simpler to make them the same?

In the meantime, I will instead revert the changes that were already made.

Reply:
@epenet Just to be complete, for service-calls the DOMAIN must be imported from the Platform.
What about when using `entity_registry.async_get_or_create()`? Can `Platform.XYZ` be used? Or same as service-call?

Reply:
They are not the same, services don't have to be bound to platforms. Services accept domains.